Source/WebCore/loader/SubresourceLoader.cpp

```diff
@@ -444,7 +444,7 @@ void SubresourceLoader::didReceiveResponse(ResourceResponse&& response, Completi
             // Existing resource is ok, just use it updating the expiration time.
             ResourceResponse revalidationResponse = response;
             revalidationResponse.setSource(ResourceResponse::Source::MemoryCacheAfterValidation);
-            resource->setResponse(WTFMove(revalidationResponse));
+            resource->setResponse(ResourceResponse { revalidationResponse });
```

Comment:
Thank you for catching this. It might be even better if we left the Move here (the "always" case), and constructed a ResourceResponse in the `!reachedTerminalState()` case:

```
if (!reachedTerminalState())
     ResourceLoader::didReceiveResponse(ResourceResponse { resource->response() }, [completionHandlerCaller = WTFMove(completionHandlerCaller)] { });
```